### PR TITLE
Update New Relic docs

### PR DESCRIPTION
**Pull request by breedx-nr**
Hi.  We'd like to see about updating these older docs to more proactively point to our dimensional metrics based registry.  Would love to know what you think and what else we can do to help get these changed.

Thanks!

### CODE REVIEW

**Comment by breedx-nr** [comment]
Just wanted to leave a gentle poke to see if anybody might be able to look at this.  Thanks.

**Comment by breedx-nr** [comment]
Bumping again.  Would it be possible to get an estimate on when we might get eyes on this?  🙏 🙇‍♂️

**Comment by breedx-nr** [comment]
Any updates?  Critiques?  Concerns?  Feedback?  🍰

File: src/docs/implementations/new-relic.adoc [hunk 1]
```diff
@@ -1,113 +1,143 @@
 = Micrometer New Relic
-Jon Schneider <jschneider@pivotal.io>
+New Relic Open Source <opensource@newrelic.com.>
 :toc:
 :sectnums:
 :system: new-relic
 
-New Relic offers a dimensional monitoring system product called Insights with a full UI and a query language called NRQL. New Relic Insights operates on a push model. Some features of NRQL assume that Insights receives a distinct event payload for every timing, count, etc. Micrometer instead ships aggregates at a prescribed interval, allowing your app's throughput to scale without concern for event propagation to Insights becoming a bottleneck.
+New Relic is a powerful observability platform with Micrometer
```
Comment (ebullient) [inline src/docs/implementations/new-relic.adoc:7]: superlatives suck. All of the platforms micrometer supports are (differently) powerful. ;)

**Comment by ebullient** [comment]
Squash down to one commit? @shakuzen and @jkschneider .. should the existing micrometer new relic extension be deprecated (and then have that reflected in the docs here?)

**Comment by breedx-nr** [comment]
Re: deprecating the existing registry -- there was some discussion a while back about New Relic possibly hosting it.  That discussion is here: https://github.com/newrelic/micrometer-registry-newrelic/issues/46.  I would love for us to collectively come to an agreement on that point. The primary concern is that users might be confused about which one to use...so having a primary recommended solution is best.

**Comment by shakuzen** [comment]
Thanks for the pull request. Sorry for the delay in responding here. I would like to have some response to https://github.com/newrelic/micrometer-registry-newrelic/issues/46#issuecomment-598586618 before making these kinds of changes to our docs. I am in support of getting to a point of there being one New Relic registry implementation that works best for users. I am concerned about making such a transition as smooth as possible, and I think that's what needs to be worked out before we officially tell users to stop using something.

**Comment by jonatan-ivanov** [comment]
I'm closing this since we moved the docs to their respective projects and https://github.com/newrelic/micrometer-registry-newrelic is deprecated wile Micrometer's New Relic registry is still supported.

**Pull request closed without merging**